tests/unit/disco/components/TestAddon.js

```diff
@@ -143,22 +141,31 @@ describe('<Addon />', () => {
       const data = { ...result, needsRestart: true, status: UNINSTALLING };
       const root = renderAddon({ addon: data, ...data });
       const restart = findDOMNode(root).querySelector('.notification.restart');
-      expect(restart.querySelector('p').textContent).toEqual('This add-on will be uninstalled after you restart Firefox.');
+
+      expect(restart.querySelector('p').textContent).toEqual(
+        'This add-on will be uninstalled after you restart Firefox.');
     });
 
     it('does not normally render a restart notification', () => {
       const root = renderAddon({ addon: result, ...result });
-      expect(findDOMNode(root).querySelector('.notification.restart')).toBeFalsy();
+
+      expect(findDOMNode(root).querySelector('.notification.restart'))
+        .toBeFalsy();
     });
 
     it('renders the heading', () => {
       const root = renderAddon({ addon: result, ...result });
-      expect(root.heading.textContent).toContain('test-heading');
+
+      expect(findDOMNode(root).querySelector('.heading').textContent)
```

Comment:
Iirc `findDOMNode` was planned to be deprecated see https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-find-dom-node.md

See also https://facebook.github.io/react/docs/react-dom.html#finddomnode re: use being discouraged.